scripts/shell/test_tools_api.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
```

Comment:
We do no write the tests in shell script. Please add unit and integration tests as done for other modules [here](https://github.com/kyma-project/kyma-companion/tree/main/tests).

Reply:
We had only two APIs and they are getting tested in this [blackbox tests](https://github.com/kyma-project/kyma-companion/tree/main/tests/blackbox). 

Now we are adding new APIs, so we can add end-to-end tests in [blackbox tests](https://github.com/kyma-project/kyma-companion/tree/main/tests/blackbox) as well using python. Maybe `tests/blackbox/src/api-tests`. We can then also run them as part of our CI/CD pipeline.

IMO, shell scripts are not scalable and if something is not in the CI then most devs won't be running them manually everytime.